pkg/cli/admin/upgrade/status/examples/4.14.1-control-plane-updated-workers-progressing.output

```diff
@@ -0,0 +1,25 @@
+An update is in progress for 2m58s: Cluster version is 4.14.1
+
+= Control Plane =
+Assessment:      Completed
+Completion:      100%
+Duration:        32m0s
```

Comment:
Yep, this looks good. I remember we fixed this inconsistency in our pair session, IIRC we showed "time since progressing changed" in SINCE, which in this phase is "when control plane finished updating".